packages/e2e-tests/specs/editor/blocks/table.test.js

```diff
@@ -24,7 +19,7 @@ const createButtonLabel = 'Create Table';
  */
 async function changeCellAlignment( align ) {
 	await clickBlockToolbarButton( 'Change column alignment' );
-	await clickButton( `Align Column ${ capitalize( align ) }` );
+	await clickButton( `Align column ${align.toLowerCase()}` );
```

Comment:
Here is a tiny linting error:

```suggestion
	await clickButton( `Align column ${ align.toLowerCase() }` );
```